Makefile

```diff
@@ -12,6 +12,11 @@ VENV_BIN_DIR := $(JSPARAGUS_DIR)jsparagus_build_venv/bin
 PYTHON := $(VENV_BIN_DIR)/python
 PIP := $(VENV_BIN_DIR)/pip
 
+init-ci:
```

Comment:
I think this make target can be renamed to `init-venv` or something, so that `init` below can depend on this,
and there won't be any duplicate.